clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -115,6 +115,10 @@ Improvements to clang-tidy
 - Improved :program:`run-clang-tidy.py` script. Fixed minor shutdown noise
   happening on certain platforms when interrupting the script.
 
+- Removed :program:`clang-tidy`'s global options for most of checks. All options
```

Comment:
Typically we are explicit about which options and which checks have been affected. It's rather verbose and painful, but currently this release note doesn't really tell much to the users.